Java8 did not start: `There is insufficient memory for the Java Runtime Environment to continue.`
Java11 had multiple issue:

Checkstyle error needs to be addressed:
```
[ant:checkstyle] [ERROR] /home/jenkins/jenkins-slave/workspace/kafka-pr-jdk11-scala2.12@2/streams/src/test/java/org/apache/kafka/streams/state/internals/TimeOrderedKeyValueBufferTest.java:42:8: Unused import - java.nio.charset.StandardCharsets. [UnusedImports]
```

Also some flaky test:
```
11:56:23 kafka.admin.ResetConsumerGroupOffsetTest > testResetOffsetsNotExistingGroup FAILED
11:56:23     java.util.concurrent.ExecutionException: org.apache.kafka.common.errors.CoordinatorNotAvailableException: The coordinator is not available.
11:56:23         at org.apache.kafka.common.internals.KafkaFutureImpl.wrapAndThrow(KafkaFutureImpl.java:45)
11:56:23         at org.apache.kafka.common.internals.KafkaFutureImpl.access$000(KafkaFutureImpl.java:32)
11:56:23         at org.apache.kafka.common.internals.KafkaFutureImpl$SingleWaiter.await(KafkaFutureImpl.java:89)
11:56:23         at org.apache.kafka.common.internals.KafkaFutureImpl.get(KafkaFutureImpl.java:262)
11:56:23         at kafka.admin.ConsumerGroupCommand$ConsumerGroupService.resetOffsets(ConsumerGroupCommand.scala:307)
11:56:23         at kafka.admin.ResetConsumerGroupOffsetTest.testResetOffsetsNotExistingGroup(ResetConsumerGroupOffsetTest.scala:89)
11:56:23 
11:56:23         Caused by:
11:56:23         org.apache.kafka.common.errors.CoordinatorNotAvailableException: The coordinator is not available.
```
and
```
12:07:33 kafka.admin.ReassignPartitionsClusterTest > shouldExecuteThrottledReassignment FAILED
12:07:33     java.lang.AssertionError: Expected replication to be < 10000 but was 11177
12:07:33         at org.junit.Assert.fail(Assert.java:88)
12:07:33         at org.junit.Assert.assertTrue(Assert.java:41)
12:07:33         at kafka.admin.ReassignPartitionsClusterTest.shouldExecuteThrottledReassignment(ReassignPartitionsClusterTest.scala:304)
```
and
```
13:24:08 kafka.api.SaslGssapiSslEndToEndAuthorizationTest > testTwoConsumersWithDifferentSaslCredentials FAILED
13:24:08     org.junit.runners.model.TestTimedOutException: test timed out after 15000 milliseconds
```
and
```
14:30:26 org.apache.kafka.streams.integration.InternalTopicIntegrationTest > shouldCompactTopicsForKeyValueStoreChangelogs FAILED
14:30:26     java.lang.RuntimeException: Timed out waiting for completion. lagMetrics=[0/2] totalLag=[0.0]
14:30:26         at org.apache.kafka.streams.integration.utils.IntegrationTestUtils.waitForCompletion(IntegrationTestUtils.java:340)
14:30:26         at org.apache.kafka.streams.integration.InternalTopicIntegrationTest.shouldCompactTopicsForKeyValueStoreChangelogs(InternalTopicIntegrationTest.java:166)
```

--- a/streams/src/test/java/org/apache/kafka/streams/state/internals/TimeOrderedKeyValueBufferTest.java
+++ b/streams/src/test/java/org/apache/kafka/streams/state/internals/TimeOrderedKeyValueBufferTest.java
@@ -39,7 +39,6 @@ import org.junit.runners.Parameterized;
 
 import java.io.IOException;
 import java.nio.ByteBuffer;
-import java.nio.charset.StandardCharsets;
 import java.util.Collection;
 import java.util.LinkedList;
 import java.util.List;

